go.mod

```diff
@@ -0,0 +1,3 @@
+module hdrhistogram
```

Comment:
The module name seems incorrect. While downloading with `go get github.com/zoidbergwill/hdrhistogram@zoid/issue-7`, I got an error: `parsing go.mod: unexpected module path "hdrhistogram"`.

The module name should be a full uri like https://github.com/caddyserver/caddy/blob/master/go.mod#L1

Reply:
Yeah, when it generated one it used "github.com/zoidbergwill/hdrhistogram"

Reply:
@zoidbergwill 
Now the `go get` works for me.